examples/python/fb15k_237_gpu.py

```diff
@@ -1,9 +1,8 @@
 from pathlib import Path
 
-from omegaconf import OmegaConf
-
 import marius as m
 from marius.tools.preprocess.datasets.fb15k_237 import FB15K237
+from omegaconf import OmegaConf
```

Comment:
are these changes necessary for the bug fix (moving the import order) or are they auto generated somehow? If it is the latter can you remove them from this PR?

Reply:
I ran `tox -e autoformat` for linting and it made those changes as I only updated the `ogbn_arxiv.py` file

Reply:
I had created an conda environment and installed tox using pip install inside of that environment. I am assuming a different version of tox got installed which lead to this different behaviour